src/main/java/message/Response.java

```diff
@@ -0,0 +1,34 @@
+package main.java.message;
+
+
+import java.io.Serializable;
+import java.util.Map;
+
+public class Response implements Serializable{
+    private int code;
+    private Map headers;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "headers" private field. ([squid:S1068](https://sonarcloud.io/organizations/jangchulwoon-github/rules#rule_key=squid%3AS1068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=groupId%3AHTTPServer&pullRequest=4&issues=AWQvx9PR_bEzQ-kUjrAs&open=AWQvx9PR_bEzQ-kUjrAs)

[ISSUE_KEY]: AWQvx9PR_bEzQ-kUjrAs